projects/packages/forms/src/blocks/contact-form/class-contact-form-block.php

```diff
@@ -800,6 +800,15 @@ public static function load_editor_scripts() {
 			),
 		);
 
+		// Preload the Forms config endpoint in the editor so apiFetch resolves synchronously.
+		add_filter(
+			'block_editor_rest_api_preload_paths',
+			static function ( $paths ) {
+				$paths[] = array( '/wp/v2/feedback/config', 'GET' );
+				return $paths;
+			}
+		);
```

Comment:
This is how we add our endpoint to the preloaded endpoints in the editor.

Reply:
I am not sure if this is working. Should we try adding this filter earlier? 

I wasn't able to see the config endpoint loaded form inline.